contrib/templates/experimental/scala-job/template/databricks.yml.tmpl

```diff
@@ -0,0 +1,52 @@
+# This is a Databricks asset bundle definition for {{.project_name}}.
+# See https://docs.databricks.com/dev-tools/bundles/index.html for documentation.
+bundle:
+  name: {{.project_name}}
+
+workspace:
+  host: {{.custom_workspace_host}}
+  artifact_path: {{.artifacts_dest_path}}/${workspace.current_user.short_name}
+
+artifacts:
+  default:
+    type: jar
+    build: sbt package && sbt assembly
+    path: .
+    files:
+      - source: {{template `jar_path` .}}
+
+resources:
+  jobs:
```

Comment:
This should be in a separate `resources/{{.project_name}}.job.yml.tmpl` file as seen in https://github.com/databricks/cli/blob/main/libs/template/templates/default-python/template/%7B%7B.project_name%7D%7D/resources/%7B%7B.project_name%7D%7D.job.yml.tmpl#L1. That folder should also have a `.gitkeep`.